velox/docs/functions/spark/datetime.rst

```diff
@@ -93,6 +93,12 @@ These functions support TIMESTAMP and DATE input types.
         SELECT from_unixtime(3600, 'yyyy'); -- '1970'
         SELECT from_unixtime(9223372036854775807, "yyyy-MM-dd HH:mm:ss");  -- '1969-12-31 23:59:59'
 
+.. spark:function:: from_utc_timestamp(timestamp, string) -> timestamp
+
+    Returns the timestamp value from UTC timezone to the given timezone.::
```

Comment:
Spark throws exception for invalid timezone id. What is the behavior for invalid timezone here, and can we add some tests for that?

Reply:
Velox throws a user exception like so
```
C++ exception with description "Exception: VeloxUserError
Error Source: USER
Error Code: INVALID_ARGUMENT
Reason: Asia/Bangalore not found in timezone database
```

Reply:
Added tests.